server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/ThreadPoolResource.java

```diff
@@ -64,11 +71,11 @@
  */
 public enum ThreadPoolResource implements ResourceDefinition, ThreadPoolDefinition {
 
-    ASYNC_OPERATIONS("async-operations", 25, 25, 1000, 60000),
-    LISTENER("listener", 1, 1, 100000, 60000),
-    REMOTE_COMMAND("remote-command", 25, 25, 100000, 60000),
-    STATE_TRANSFER("state-transfer", 1, 60, 0, 60000),
-    TRANSPORT("transport", 25, 25, 100000, 60000),
+    ASYNC_OPERATIONS("async-operations", 1, getDefaultThreads(ASYNC_OPERATIONS_EXECUTOR), 1000, 60000),
+    LISTENER("listener", 1, getDefaultThreads(ASYNC_NOTIFICATION_EXECUTOR), 0, 60000),
+    REMOTE_COMMAND("remote-command", getDefaultThreads(REMOTE_COMMAND_EXECUTOR), getDefaultThreads(REMOTE_COMMAND_EXECUTOR), 100000, 60000),
+    STATE_TRANSFER("state-transfer", 1, getDefaultThreads(STATE_TRANSFER_EXECUTOR), 0, 60000),
+    TRANSPORT("transport", getDefaultThreads(ASYNC_TRANSPORT_EXECUTOR), getDefaultThreads(ASYNC_TRANSPORT_EXECUTOR), 100000, 60000),
```

Comment:
Huh, I thought this would be much harder :)

Please move this to a separate commit `ISPN-10040 Embedded and server thread pool defaults should be the same`

Reply:
Sure

Reply:
created https://github.com/infinispan/infinispan/pull/6788